README.md

```diff
@@ -95,13 +95,31 @@ npm install
 
 ### Linux
 
-Install [Node.js][Node.js] v16. Then you can install dependencies with:
+Ensure you have the following installed:
+
+- [Node.js][Node.js] v16. **Make sure you have any development packages
+  installed.** For example, on openSUSE Leap 15.3 you would need to install
+  `nodejs16` and `nodejs16-devel`.
+
+- Go 1.18 or later.
+
+- Dependencies described in the [`node-gyp` docs][node-gyp] installation.
+  This is required to install the [`ffi-napi`][ffi-napi] npm package. These docs mention
+  "a proper C/C++ compiler toolchain". You can install `gcc` and `g++` for this.
+
+Then you can install dependencies with:
 
 ```
 npm install
 ```
 
+You can then run Rancher Desktop as described below. It may fail on the first run -
```

Comment:
More details on the specific failure may be helpful (to ensure we don't end up with _other_ failures we don't know about).